music_server.py

```diff
@@ -16,4 +39,7 @@ def page_not_found(e):
 
 if __name__ == '__main__':
 	logging.getLogger('waitress').setLevel(logging.DEBUG)
-	serve(app, host='0.0.0.0', port=9999)
\ No newline at end of file
+	serve(app, host='0.0.0.0', port=9999)
+	#print(f"Check path : {os.path.exists(SEARCH_FOLDER_PATH)}")
```

Comment:
I assume that this line was used for debugging? I assume you do not need it anymore, so could you please remove it?

Reply:
Yes lol. Is/was useful when starting to debug and have it running

Reply:
Yup! That makes sense! 😆